static/css/section/_homepage.scss

```diff
@@ -141,75 +141,74 @@ body.homepage {
   }
 }
 
-.row--intro {
-  background: $ubuntu-orange;
-  color: #fff;
-}
+// row--intro / ubuntu-intro
+$full-size: 1020px;
```

Comment:
@WillMoggridge it's a very specific measurement to this component. I think that it's cleaner here.